lib/components/Pivot/Pivot.tsx

```diff
@@ -1,11 +1,9 @@
 import * as React from 'react';
 import * as classNames from 'classnames/bind';
-import styled, { ThemeProps } from 'styled-components';
 
 import {DivProps, SpanProps, Elements as Attr} from '../../Attributes';
 import {MethodNode} from '../../Common';
 import {Icon, IconSize, IconAttributes} from '../Icon';
-import { ShellTheme } from '../Shell';
 
 const css = classNames.bind(require('./Pivot.module.scss'));
```

Comment:
probably in this lib we can start to require the scss file avoiding to use classNames since we don't have multiple classes to use.

Reply:
Yeah we could, would we get mangling without classNames?